app/views/products/_product.html.erb

```diff
@@ -14,5 +14,10 @@
       <h5><%= number_to_currency(product.price) %></h5>
       <p class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Amet numquam aspernatur!</p>
     </div>
+    <% if current_admin_user %>
+      <div class="card-footer">
+        <%= button_to "Delete", admin_product_path(product), method: :delete, class: "btn btn-danger"%>
+      </div>
+    <% end %>
```

Comment:
Tu się zastanawiałem jak to rozwiązać, bo to wspólny partial dla usera i dla admina. To wydawało mi się najprostsze, ale myślałem też, żeby do partiala przekazać bezbośrednio jakąś lokalną zmienną boolean. Wtedy jeśli tego partiala użyję w innym widoku,  to mógłbym wybrać czy wyświetlać ten button.

Reply:
nie komplikowalbym partiali :) zrob drugi